### PR TITLE
[PKG-1483] Updates nbclassic to 0.5.4

### PR DESCRIPTION
**Pull request by schuylermartin45**
- Updates `nbclassic` to `v0.5.4`
- Local build and linter passes

### CODE REVIEW

**Review by ELundby45 — APPROVED** [review]
LGTM!

**Review by varlackc — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,14 @@
 {% set name = "nbclassic" %}
-{% set version = "0.5.3" %}
+{% set version = "0.5.4" %}
+{% set sha256 = "312b3f7d7ff2e6c261d51799bb12e6493498ab47d3469d3a01015d5533fd4d2b" %}
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:3]: There is no need for the jinja2 variable if it's used only once. The recipe now has two lines about sha256 instead of one line

Comment (schuylermartin45) [reply recipe/meta.yaml:3]: I've taken this as an action in "future-proofing"/standardizing as I've seen several instances where the variable was defined only refer to the variable once and do this. I assumed this was a preferred mechanism that we were all moving towards.

If it is something people have strong opinions on, maybe the linter needs a new rule(?).

**Pull request merged**